doc/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # pip doesn't handle well dependencies with a relative path inside a requirements.txt file.
 # If we put .. instead of 'openfisca_core >= 4.2.0, < 5.0', it will look for a setup.py in the parent directory relatively to where the instruction is executed from, and not where the requirements file is.
-openfisca_core >= 4.2.0, < 6.0
+openfisca_core >= 9.0.0
```

Comment:
This is the version to be used in the [reference doc](http://openfisca.readthedocs.io/en/latest/). We want to take the last published.